db/schema.rb

```diff
@@ -15,18 +15,6 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "admins", force: :cascade do |t|
-    t.string "email", default: "", null: false
-    t.string "encrypted_password", default: "", null: false
-    t.string "reset_password_token"
-    t.datetime "reset_password_sent_at"
-    t.datetime "remember_created_at"
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
-    t.index ["email"], name: "index_admins_on_email", unique: true
-    t.index ["reset_password_token"], name: "index_admins_on_reset_password_token", unique: true
-  end
```

Comment:
Did you manually remove this table?
Was there any migration to do this?

Reply:
Hi Eduardo, this by mistake went to the master, it should not be there, there is no such model.

Reply:
If not, I'd suggest using a migration instead of manually editing the `schema`

Reply:
Thank you for your advice, we will take it into account so as not to fall into the same error.